docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>We should explain the transformed value variables. I think the original doc we are removing also did thay. May be nice to first show a model without transformed variables and then one with them. I emphasize this because it's a common source of confusion </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='95'/>

Reply:
Sure! I will keep this one in mind once I get to this section :)